AIO script improvements (#602)
stackhpc/stackhpc-kayobe-config: doc/source/contributor/environments/ci-aio.rst

   ./automated-setup.sh

> Adding the ``--lvm`` flag will also automatically set the LVM configuration,

Reviewer: ```suggestion
Unless the ``--non-lvm`` flag is set, LVM configuration is automatically applied,
```
